libs/designsystem/src/lib/components/dropdown/dropdown.component.ts

```diff
@@ -60,6 +60,7 @@ export class DropdownComponent
   private hasConfiguredSlottedItems = false;
   private horizontalDirection = HorizontalDirection.right;
   private verticalDirection = VerticalDirection.down;
+  private _forcePopoutVerticalDirection: VerticalDirection = null;
```

Comment:
Not sure there's a specific reason 😁. Maybe because I prefer to be explicit about the direction. It's neither `up` or `down`.

Anyway, I've replaced the property with a boolean `disableAutoVerticalDirection` so it's not an issue any more 😊